sql/core/src/test/resources/sql-tests/inputs/pipe-operators.sql

```diff
@@ -665,15 +665,119 @@ table t
 table t
 |> order by x sort by x;
 
--- The WINDOW clause is not supported yet.
+-- The WINDOW clause: positive tests.
+-------------------------------------
+-- WINDOW with one definition
 table windowTestData
-|> window w as (partition by cte order by val)
+|> window w as (partition by cate order by val)
 |> select cate, sum(val) over w;
 
+-- WINDOW with multiple definitions
+table windowTestData
+|> window w1 as (partition by cate), w2 as (partition by val order by cate)
+|> select sum(val) over w1, first_value(cate) over w2;
+
+-- WINDOW with RANGE BETWEEN as part of the window definition
```

Comment:
silly style nitpick: can we add a period to the end of each of these comments just to be consistent with the other similar comments in this file :)